On NetBSD, work around multibyte bug detected by gsed 4.10 (bug standards/60369)

--- buildsystems/autotools/meson.build.orig	2026-04-10 08:54:24 UTC
+++ buildsystems/autotools/meson.build
@@ -22,6 +22,7 @@ foreach mode : ['', 'no-xslt']
     install: true,
     install_dir: pkgdatadir,
     capture: true,
+    env: {'LC_ALL': 'C'},
     command: [
       'sed',
       '-e',
